packages/roosterjs-editor-plugins/lib/plugins/Paste/Paste.ts

```diff
@@ -109,6 +111,8 @@ export default class Paste implements EditorPlugin {
                 clipboardData.htmlFirstLevelChildTags[0] == 'IMG'
             ) {
                 convertPasteContentForSingleImage(event, trustedHTMLHandler);
+            } else if (isWordOnlineTables(fragment)) {
```

Comment:
What if the copied content contains both table and list? Will it fall into line 91?

We need to run both list and table process when copy from word only, but not a "else if" condition.